src/legacy/core_plugins/console/server/api_server/spec/generated/create.json

```diff
@@ -22,8 +22,7 @@
       "POST"
     ],
     "patterns": [
-      "{indices}/_create/{id}",
-      "{indices}/{type}/{id}/_create"
+      "{indices}/_create/{id}"
```

Comment:
I haven't been able to generate this one

<img width="494" alt="Screen Shot 2020-01-14 at 16 51 45" src="https://user-images.githubusercontent.com/2854616/72340526-60100380-36ee-11ea-9377-93961f273238.png">

Reply:
Will make an issue for tracking!

Reply:
Nvm, was easy enough to fix :+1:

<img width="379" alt="Screenshot 2020-01-14 at 17 28 43" src="https://user-images.githubusercontent.com/8155004/72362393-6359be00-36f3-11ea-9ab4-1b35164224c7.png">